### ISSUE TITLE
[Suggestion] Void Essense uses

### ISSUE THREAD

**Issue by VallenFrostweaver** [RootsTeam/Animunda#7]
Perhaps you can use Void essense to repair the Void tools/weapons? A spell caster option for sure to repair void items on hot bar or armor that is being worn - perhaps Void Armor? :D

**Comment by ShadowGamerXY** [comment]
Yeah in the upcoming version is there is an experimental void infusing system which infuses your tools, later on armor as well to have different special abilities(at a cost of void essence or something else) and there is also planned to have more tiers/ranks of infusion that will make your tools/armor more powerful(not op, and there will be appropriate cost as well)
